README.md

```diff
@@ -25,8 +25,62 @@
 - **tmp_path**: temporary file directory. If null, it is associated with the default FileSystem. (string, default: null)
 - **tmp_path_prefix**: prefix of temporary files (string, default: 'embulk-output-s3-')
 - **canned_acl**: canned access control list for created objects ([enum](#cannedaccesscontrollist), default: null)
-- **proxy_host**: proxy host to use when accessing AWS S3 via proxy. (string, default: null )
-- **proxy_port**: proxy port to use when accessing AWS S3 via proxy. (string, default: null )
+- [Deprecated] **proxy_host**: proxy host to use when accessing AWS S3 via proxy. (string, default: null )
+- [Deprecated] **proxy_port**: proxy port to use when accessing AWS S3 via proxy. (string, default: null )
+- **http_proxy** http proxy configuration to use when accessing AWS S3 via http proxy. (optional)
+  - **host** proxy host (string, required)
+  - **port** proxy port (int, optional)
+  - **https** use https or not (boolean, default true)
+  - **user** proxy user (string, optional)
+  - **password** proxy password (string, optional)
+
```

Comment:
nit: Add `:`?  (ex: **password**:)

Reply:
> nit: Add :? (ex: password:)
Cool. (I just copied from README.md in s3 input.

Reply:
I changed PR subject.

Reply:
[383e1a8](https://github.com/embulk/embulk-output-s3/pull/30/commits/383e1a80aa5ab763a9bf3704715a9d496f329ebd)

> nit: Add :? (ex: password:)

done